docs/Customs Exports Internal AutoComplete.js

```diff
@@ -85,14 +85,14 @@ function completePage() {
         validDate.setDate(validDate.getDate() - 1); // One day before
         if (title.indexOf('departure') !== -1) {
             document.getElementById('dateOfDeparture_day').value = validDate.getDate();
-            document.getElementById('dateOfDeparture_month').value = validDate.getMonth();
+            document.getElementById('dateOfDeparture_month').value = validDate.getMonth()+1;
```

Comment:
Is this change just because `getMonth()` returns "0" for January?

Reply:
Yes.